client/main.js

```diff
@@ -1,15 +1,23 @@
-const width = 1872;
-const height = 1404;
-
-const rawCanvas = new OffscreenCanvas(width, height); // Define width and height as needed
+const rawCanvas = new OffscreenCanvas(screenWidth, screenHeight); // Define width and height as needed
 let portrait = getQueryParam('portrait');
 portrait = portrait !== null ? portrait === 'true' : false;
-let flip = getQueryParam('flip');
-flip = flip !== null ? flip === 'true' : false;
+
+defaultFlip = true;
```

Comment:
On RM2 the defaultFlip should be false as well (otherwise the image is upside down)

Reply:
I will change it